scripts/generateSchemas.ts

```diff
@@ -53,17 +59,18 @@ function writeFile(path: string, data: string) {
   writeFileSync(path, data, "utf8");
 }
 
-function sanitiseFolders(): void {
+function sanitiseFolders() {
+  try {
+    sanitiseTypesFolder(typesDirName);
+  } catch {}
```

Comment:
Wouldn't you want the catch to log the error?

Reply:
Well the error would be for instance, trying to delete a file that doesn't exist, which doesn't really necessitate logging.